about.html

```diff
@@ -0,0 +1,110 @@
+        <li><a href="#" class="navLink">News</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class="hamburger flex">
+    <div></div>
+    <div></div>
+    <div></div>
+  </div>
+
+  <main>
+    <section class="about-intro flex">
+      <div class="intro-heading flex about-intro-heading">
+        <h2>"Hello! EDM Lovers"</h2>
+        <h1>Electronic Music Global Summit 2022</h1>
+      </div>
+      <p class="mobile-paragraph about-mobile-paragraph">This EDM Global Summit brings together people who are
+        passionate about creating EDM Music, Parties! You dance, You rock, You get drunk! Everything you could wish for
+        is here! Our Featured Speakers are world's top DJ's and Record Producers. Think of this conference as a way to
+        learn, connect and enjoy.</p>
+      <p class="about-Email-paragraph">Please contact us per Email for any further questions about EDM Global Summit!
+      </p>
+      <a class="about-Email-paragraph about-Email" href="mailto:contactus@edmsummit.org">contactus@edmsummit.org</a>
+    </section>
+  </main>
+
+  <section class="edmLogo flex">
+    <h2>Electronic Music Global Summit 20222 Logo</h2>
+    <hr>
+    <p class="about-Email-paragraph">The Logo of Electronic Music Global Summit was designed by Sam Felice.</p>
+    <div class="aboutEdmLogo flex">
+      <img src="./capstone_assets/LOGO.png" alt="EDM Global Summit Logo">
+    </div>
+  </section>
+
+  <section class="edmLogo flex">
+    <h2>See the past Electronic Music Global Summit</h2>
+    <hr>
+    <p class="about-Email-paragraph">Take a look at past two EDM Global Summit which took place in Belgium and HongKong.
+    </p>
+
+    <div class="summit1">
+      <div class="summit-overlay"></div>
+      <section class="Summit-text">
+        <h2>2021</h2>
+        <span>EDM Global Summit 2021 in Belgium</span>
+      </section>
+    </div>
+
+    <div class="summit2">
+      <div class="summit-overlay"></div>
+      <section class="Summit-text">
+        <h2>2019</h2>
+        <span>EDM Global Summit 2019 in HongKong</span>
+      </section>
+    </div>
+  </section>
+
+  <footer>
+    <div class="footerHeader flex">
+      <h2>Partner</h2>
+      <hr>
+    </div>
+    <div class="partnerIcons">
+      <div class="flex imgone">
+        <img class="partnerone" src="./capstone_assets/STMPD.png" alt="STMPD Records Logo">
+      </div>
+      <div class="flex imgone">
+        <img class="partnertwo" src="./capstone_assets/Tomorrowland.png" alt="Tomorrowland Logo">
+      </div>
+      <div class="flex imgone">
+        <img class="partnerthree" src="./capstone_assets/coachella.png" alt="Coachella Logo">
+      </div>
+    </div>
+
+    <section class="Footerlogo flex">
+      <a href="./index.html"><img src="./capstone_assets/LOGO.png" alt="EDM Summit Logo"></a>
+      <div class="copyrights">
+        <p>2022 Created by Samuel Felice</p>
+        <p>Original Design by Cindy Shin</p>
+      </div>
+    </section>
+  </footer>
```

Comment:
- [ ] There is no partner's section on the desktop version of the about page, just a dark footer. Kindly address this issue so as to follow the design template 👍 
- Your design:
![Annotation 2022-05-25 151644](https://user-images.githubusercontent.com/85113567/170286357-96c1b95b-d15f-4421-9e80-4d2ef609ad34.jpg)
- Required design:
![Annotation 2022-05-25 151704](https://user-images.githubusercontent.com/85113567/170286395-3b5f9af7-937b-4768-980c-9d2c79c9f70a.jpg)